Mathlib/Data/MvPolynomial/Supported.lean

```diff
@@ -60,15 +60,15 @@ noncomputable def supportedEquivMvPolynomial (s : Set σ) : supported R s ≃ₐ
     (AlgEquiv.ofInjective (rename ((↑) : s → σ)) (rename_injective _ Subtype.val_injective)).symm
 #align mv_polynomial.supported_equiv_mv_polynomial MvPolynomial.supportedEquivMvPolynomial
 
-@[simp]
+--@[simp]  -- the `simpNF` linter complained about this lemma.
```

Comment:
Comments like this should include the words `porting note:`

Reply:
Added `porting note:`s!